src/components/auth/RecoverPassword.vue

```diff
@@ -48,6 +62,23 @@ export default class RecoverPassword extends Vue {
     ];
     this.submitText = this.$t('authPages.recoverPassword');
   }
+
+  /**
+   * Method recover user's password by email from form
+   */
+  private async recoverPassword(): Promise<void> {
+    const email = this.fields[0].value;
+
+    try {
+      await userApi.recoverPassword(email);
+      this.$router.push('/login');
```

Comment:
how I will know the new password?

Reply:
The new password will be sent to the email